db/db_impl_open.cc

```diff
@@ -124,6 +124,18 @@ DBOptions SanitizeOptions(const std::string& dbname, const DBOptions& src) {
     result.avoid_flush_during_recovery = false;
   }
 
+#ifndef ROCKSDB_LITE
+  // When the DB is stopped, it's possible that there are some .trash files that
+  // were not deleted yet, when we open the DB we will find these .trash files
+  // and schedule them to be deleted
+  auto sfm = static_cast<SstFileManagerImpl*>(result.sst_file_manager.get());
+  if (sfm) {
```

Comment:
If sfm is not empty, it may be a good idea to still immediately delete those .trash files.

Reply:
Sounds good, let me do that